### PR TITLE
feat: support localization

### PR DESCRIPTION
**Pull request by kkumar-gcc**
This PR adds the ability for users to switch between languages. Currently, the system only supports two languages: English (`en`) and Chinese (`zh_CN`). Users can set their preferred language by using the `APP_LOCALE` environment variable or by modifying the `config/app.go` file.

### CODE REVIEW

File: app/console/commands/monitoring.go [hunk 1]
```diff
@@ -27,7 +28,8 @@ func (receiver *Monitoring) Signature() string {
 
 // Description The console command description.
 func (receiver *Monitoring) Description() string {
-	return "[面板] 系统监控"
+	ctx := context.Background()
+	return facades.Lang(ctx).Get("panel:monitoring.description")
```
Comment (devhaozi) [inline app/console/commands/monitoring.go:32]: suggest use: `panel:command.monitoring.description`

File: app/console/commands/panel.go [hunk 2]
```diff
@@ -31,7 +32,8 @@ func (receiver *Panel) Signature() string {
 
 // Description The console command description.
 func (receiver *Panel) Description() string {
-	return "[面板] 命令行"
+	ctx := context.Background()
+	return facades.Lang(ctx).Get("panel.description")
```
Comment (devhaozi) [inline app/console/commands/panel.go:36]: suggest use: `panel:command.panel.description`

File: app/console/commands/panel.go [hunk 3]
```diff
@@ -159,10 +163,10 @@ func (receiver *Panel) Handle(ctx console.Context) error {
 			return nil
 		}
 
-		color.Greenln("面板端口: " + port)
+		color.Greenln(translate.Get("panel.getPort.port") + ": " + port)
```
Comment (devhaozi) [inline app/console/commands/panel.go:166]: suggest use: `panel:command.panel.getPort`

File: app/console/commands/panel.go [hunk 4]
```diff
@@ -159,10 +163,10 @@ func (receiver *Panel) Handle(ctx console.Context) error {
...
 
 	case "getEntrance":
-		color.Greenln("面板入口: " + facades.Config().GetString("http.entrance"))
+		color.Greenln(translate.Get("panel.getEntrance.entrance") + ": " + facades.Config().GetString("http.entrance"))
```
Comment (devhaozi) [inline app/console/commands/panel.go:169]: suggest use: `panel:command.panel.getEntrance`

File: app/console/commands/panel.go [hunk 5]
```diff
@@ -687,31 +691,31 @@ func (receiver *Panel) Handle(ctx console.Context) error {
...
-		color.Yellowln("panel init 初始化面板")
-		color.Yellowln("panel writePlugin {slug} {version} 写入插件安装状态")
-		color.Yellowln("panel deletePlugin {slug} 移除插件安装状态")
-		color.Yellowln("panel writeMysqlPassword {password} 写入MySQL root密码")
-		color.Yellowln("panel writeSite {name} {status} {path} {php} {ssl} 写入网站数据到面板")
-		color.Yellowln("panel deleteSite {name} 删除面板网站数据")
-		color.Yellowln("panel getSetting {name} 获取面板设置数据")
-		color.Yellowln("panel writeSetting {name} {value} 写入 / 更新面板设置数据")
-		color.Yellowln("panel deleteSetting {name} 删除面板设置数据")
+		color.Yellowln(facades.Config().GetString("panel.name") + " - " + translate.Get("panel.cmd_tool") + " - " + facades.Config().GetString("panel.version"))
```
Comment (devhaozi) [inline app/console/commands/panel.go:694]: suggest use: `panel:command.panel.tool`

File: app/console/commands/panel.go [hunk 6]
```diff
@@ -687,31 +691,31 @@ func (receiver *Panel) Handle(ctx console.Context) error {
...
+		color.Greenln(translate.Get("panel.use_commands") + "：")
```
Comment (devhaozi) [inline app/console/commands/panel.go:695]: suggest use: `panel:command.panel.use`

File: app/console/commands/panel.go [hunk 7]
```diff
@@ -687,31 +691,31 @@ func (receiver *Panel) Handle(ctx console.Context) error {
...
+		color.Greenln("panel update " + translate.Get("panel.update.description"))
```
Comment (devhaozi) [inline app/console/commands/panel.go:696]: suggest use: `panel:command.panel.update`

File: app/console/commands/panel.go [hunk 8]
```diff
@@ -687,31 +691,31 @@ func (receiver *Panel) Handle(ctx console.Context) error {
...
+		color.Greenln("panel getInfo " + translate.Get("panel.getInfo.description"))
```
Comment (devhaozi) [inline app/console/commands/panel.go:697]: suggest use: `panel:command.panel.getInfo`

File: app/console/commands/panel.go [hunk 9]
```diff
@@ -687,31 +691,31 @@ func (receiver *Panel) Handle(ctx console.Context) error {
...
+		color.Greenln("panel getPort " + translate.Get("panel.getPort.description"))
```
Comment (devhaozi) [inline app/console/commands/panel.go:698]: suggest use: `panel:command.panel.getPort`

File: app/console/commands/panel.go [hunk 10]
```diff
@@ -687,31 +691,31 @@ func (receiver *Panel) Handle(ctx console.Context) error {
...
+		color.Greenln("panel getEntrance " + translate.Get("panel.getEntrance.description"))
```
Comment (devhaozi) [inline app/console/commands/panel.go:699]: suggest use: `panel:command.panel.getEntrance`

File: app/console/commands/panel.go [hunk 11]
```diff
@@ -687,31 +691,31 @@ func (receiver *Panel) Handle(ctx console.Context) error {
...
+		color.Greenln("panel deleteEntrance " + translate.Get("panel.deleteEntrance.description"))
```
Comment (devhaozi) [inline app/console/commands/panel.go:700]: suggest use: `panel:command.panel.deleteEntrance`

File: app/console/commands/panel.go [hunk 12]
```diff
@@ -687,31 +691,31 @@ func (receiver *Panel) Handle(ctx console.Context) error {
...
+		color.Greenln("panel cleanTask " + translate.Get("panel.cleanTask.description"))
```
Comment (devhaozi) [inline app/console/commands/panel.go:701]: suggest use: `panel:command.panel.cleanTask`

File: app/console/commands/panel.go [hunk 13]
```diff
@@ -687,31 +691,31 @@ func (receiver *Panel) Handle(ctx console.Context) error {
...
+		color.Greenln("panel backup {website/mysql/postgresql} {name} {path} {save_copies} " + translate.Get("panel.backup.description"))
```
Comment (devhaozi) [inline app/console/commands/panel.go:702]: suggest use: `panel:command.panel.backup`

File: app/console/commands/panel.go [hunk 14]
```diff
@@ -687,31 +691,31 @@ func (receiver *Panel) Handle(ctx console.Context) error {
...
+		color.Greenln("panel cutoff {website_name} {save_copies} " + translate.Get("panel.cutoff.description"))
```
Comment (devhaozi) [inline app/console/commands/panel.go:703]: suggest use: `panel:command.panel.cutoff`

File: app/console/commands/panel.go [hunk 15]
```diff
@@ -687,31 +691,31 @@ func (receiver *Panel) Handle(ctx console.Context) error {
...
+		color.Greenln("panel installPlugin {slug} " + translate.Get("panel.installPlugin.description"))
```
Comment (devhaozi) [inline app/console/commands/panel.go:704]: suggest use: `panel:command.panel.installPlugin`

File: app/console/commands/panel.go [hunk 16]
```diff
@@ -687,31 +691,31 @@ func (receiver *Panel) Handle(ctx console.Context) error {
...
+		color.Greenln("panel uninstallPlugin {slug} " + translate.Get("panel.uninstallPlugin.description"))
+		color.Greenln("panel updatePlugin {slug} " + translate.Get("panel.updatePlugin.description"))
+		color.Greenln("panel addSite {name} {domain} {port} {path} {php} " + translate.Get("panel.addSite.description"))
+		color.Greenln("panel removeSite {name} " + translate.Get("panel.removeSite.description"))
```
Comment (devhaozi) [inline app/console/commands/panel.go:708]: suggest use: `panel:command.panel.removeSite`

File: app/console/commands/panel.go [hunk 17]
```diff
@@ -687,31 +691,31 @@ func (receiver *Panel) Handle(ctx console.Context) error {
...
+		color.Redln(translate.Get("panel.developer_guidance") + ":")
+		color.Yellowln("panel init " + translate.Get("panel.init.description"))
+		color.Yellowln("panel writePlugin {slug} {version} " + translate.Get("panel.writePlugin.description"))
+		color.Yellowln("panel deletePlugin {slug} " + translate.Get("panel.deletePlugin.description"))
+		color.Yellowln("panel writeMysqlPassword {password} " + translate.Get("panel.writeMysqlPassword.description"))
```
Comment (devhaozi) [inline app/console/commands/panel.go:713]: suggest use: `panel:command.panel.writeMysqlPassword`

File: app/console/commands/panel.go [hunk 18]
```diff
@@ -687,31 +691,31 @@ func (receiver *Panel) Handle(ctx console.Context) error {
...
+		color.Yellowln("panel writeSite {name} {status} {path} {php} {ssl} " + translate.Get("panel.writeSite.description"))
```
Comment (devhaozi) [inline app/console/commands/panel.go:714]: suggest use: `panel:command.panel.writeSite`

File: app/console/commands/panel.go [hunk 19]
```diff
@@ -687,31 +691,31 @@ func (receiver *Panel) Handle(ctx console.Context) error {
...
+		color.Yellowln("panel deleteSite {name} " + translate.Get("panel.deleteSite.description"))
```
Comment (devhaozi) [inline app/console/commands/panel.go:715]: suggest use: `panel:command.panel.deleteSite`

File: app/console/commands/panel.go [hunk 20]
```diff
@@ -687,31 +691,31 @@ func (receiver *Panel) Handle(ctx console.Context) error {
...
+		color.Yellowln("panel getSetting {name} " + translate.Get("panel.getSetting.description"))
```
Comment (devhaozi) [inline app/console/commands/panel.go:716]: suggest use: `panel:command.panel.getSetting`

File: app/console/commands/panel.go [hunk 21]
```diff
@@ -687,31 +691,31 @@ func (receiver *Panel) Handle(ctx console.Context) error {
...
+		color.Yellowln("panel writeSetting {name} {value} " + translate.Get("panel.writeSetting.description"))
```
Comment (devhaozi) [inline app/console/commands/panel.go:717]: suggest use: `panel:command.panel.writeSetting`

File: app/console/commands/panel.go [hunk 22]
```diff
@@ -687,31 +691,31 @@ func (receiver *Panel) Handle(ctx console.Context) error {
...
+		color.Yellowln("panel deleteSetting {name} " + translate.Get("panel.deleteSetting.description"))
```
Comment (devhaozi) [inline app/console/commands/panel.go:718]: suggest use: `panel:command.panel.deleteSetting`

File: app/console/commands/panel_task.go [hunk 23]
```diff
@@ -21,7 +23,7 @@ func (receiver *PanelTask) Signature() string {
 
 // Description The console command description.
 func (receiver *PanelTask) Description() string {
-	return "[面板] 每日任务"
+	return facades.Lang(context.Background()).Get("panel:task.description")
```
Comment (devhaozi) [inline app/console/commands/panel_task.go:26]: suggest use: `panel:command.task.description`

File: lang/zh_CN.json
Comment (devhaozi) [inline lang/zh_CN.json:1]: Sync with above

File: lang/en.json
Comment (devhaozi) [inline lang/en.json:1]: Sync with above

**Review by devhaozi — CHANGES_REQUESTED** [review]

**Comment by kkumar-gcc** [comment]
Translation are organized as 
```json
"commands": {
   "command:signature": {
     ...
     "sub_command:signature": {...}
   }
}
```

**Review by devhaozi — APPROVED** [review]
LGTM

**Review by devhaozi — APPROVED** [review]
LGTM

**Pull request merged**